skunkworks_crow/src/main/java/org/odk/share/views/ui/bluetooth/BtReceiverActivity.java

```diff
@@ -311,4 +320,24 @@ protected void onDestroy() {
         super.onDestroy();
         unregisterReceiver(bluetoothReceiver);
     }
+
+    @Override
+    public void onBackPressed() {
+        if (BluetoothUtils.isBluetoothEnabled()) {
```

Comment:
What if bluetooth is disabled by user from the `Device notification drawer`, user won't be able to navigate back.

Reply:
fixed